types/react-native/index.d.ts

```diff
@@ -4269,6 +4269,8 @@ export interface SectionListProps<ItemT> extends VirtualizedListWithoutRenderIte
      * Only enabled by default on iOS because that is the platform standard there.
      */
     stickySectionHeadersEnabled?: boolean;
+    
+    legacyImplementation?: boolean;
```

Comment:
Is there any documentation you could add to what this is for?

Reply:
To be honest, I'm not really sure what this property does, nor have I found it anywhere in the official docs. 
The only thing I know is that settings this property to true fixed a data truncating issue with SectionList on Android 4.4. 

I've seen this property used in typings for FlatList, also without docs.

Reply:
Ok, I've had a look into the source code, I'm gonna update the docs.

Reply:
Great, thanks!

You may also want consider submitting those docs upstream.

Reply:
Good point :)